plugin/storage/cassandra/schema/create.sh

```diff
@@ -30,9 +31,19 @@ if [[ "$MODE" == "" ]]; then
     usage "missing MODE parameter"
 elif [[ "$MODE" == "prod" ]]; then
     if [[ "$DATACENTER" == "" ]]; then usage "missing DATACENTER parameter for prod mode"; fi
+    replication_factor=${REPLICATION_FACTOR:-3}
     datacenter=$DATACENTER
-    replication_factor=${REPLICATION_FACTOR:-2}
-    replication="{'class': 'NetworkTopologyStrategy', '$datacenter': '${replication_factor}' }"
+    topology_config=""
+    count=0
+    for dc in $DATACENTER; do
+      if [[ "$count" -eq "0" ]]; then
+        topology_config="'$dc': '${replication_factor}'"
+      else
+        topology_config="$topology_config, '$dc': '${replication_factor}'"
+      fi
+      count=$((count+1))
+    done
+    replication="{'class': 'NetworkTopologyStrategy', ${topology_config}}"
```

Comment:
For more complex multi dc replication strategies does it make sense to just allow the operator to override the entire replication string using an environment variable? 

If we add this then eventually someone will want datacenters with different RF values.

Reply:
@joe-elliott @yurishkuro I agree. I think having the entire string would be simple from an implementation strategy, but possibly a bit harder to debug errors for users who are have little Cassandra experience, or deploying at the operator. I think there are multiple options here:

1. Provide the entire replication config - this would be the easiest to implement, but would be pretty awful to put into a yaml file for the operator config: 
```
{ 'class': 'NetworkTopologyStrategy', 'dc1': '1', 'dc2': '2' ... }
```
2. Provide just the data-center portion, and still use the `mode` setting - this is less offensive to a user of the operator, and pretty simple to validate/implement:
```
'dc1': '1', 'dc2': '2', ...
```
3. Provide a parseable format, such as DC1:RF1,DC2:RF2 - probably the simplest on all fronts, development and using it in yaml, but it would require informing the users specifically what it is:
```
dc1:1,dc2:2,...
```
I would suggest option 3, but I come from the inexperienced with Cassandra camp, and don't know if this would be confusing and hard to understand. I don't fully believe so. Another negative, though, would be losing backwards compatibility with previous versions of the operator, as I would expect a `replicationFactorConfig` value of sorts to be introduced. Obviously there are ways to provide both and bark at the user if both are provided, but I will leave that up for discussion.

Let me know what y'all think!

Reply:
I like the idea of just providing a full replacement string for replication config, instead of building complicated piecemeal support.

Reply:
@yurishkuro @joe-elliott Alright! Just want to be clear, that preference is for Option 1 above? I can update this PR and start getting to work on the Operator changes. I may need some help on that side, as my golang is pretty elementary, but I'm sure I can get things started.

Reply:
Yes, let's do option 1.

Reply:
OK - my goal will be to leave the `$DATACENTER` usage as previously provided, and add support for `$REPLICATION_FACTOR_CONFIG`, that when provided, will override the creation of the `replication_factor` with the provided value. Will also PR the Operator. Thanks for the input guys, will circle back over the next few days as I get time to work on this.